index.js

```diff
@@ -20,6 +20,36 @@ StylusFilter.prototype.targetExtension = 'css';
 
 StylusFilter.prototype.processString = function (str) {
 	var opts = this.options;
+
+	var importRegex = new RegExp(/@import ['"].*['"]/);
+	while (importRegex.test(str)) {
+		var imp = importRegex.exec(str);
+		var sourceReg = new RegExp(/['"].*['"]/);
+		var extReg = new RegExp(/.styl/);
```

Comment:
The usage of `.` here is incorrect. If you want to match `.styl`, the dot must be escaped.